decidim-core/app/views/decidim/errors/internal_server_error.html.erb

```diff
@@ -4,7 +4,41 @@
     <div class="static__message">
       <h1 class="heading1 page-title"><%= t(".title") %></h1>
       <p><%= t(".try_later") %></p>
+      <div rows="6" id="input-text">
+        <div class="card--list__item">
+          <div class="card--list__text">
+            <strong class="margin-right-2"><%= t(".user_id") %></strong>
+            <%= user = try(:current_user)&.id || t(".unknown") %>
+          </div>
+        </div>
+        <div class="card--list__item">
+          <div class="card--list__text">
+            <strong class="margin-right-2"><%= t(".date_and_time") %></strong>
+            <%= date_and_time = l(Time.current, format: "%d/%m/%Y - %H:%M") %>
+          </div>
+        </div>
+        <div class="card--list__item">
+          <div class="card--list__text">
+            <strong class="margin-right-2"><%= t(".url") %></strong>
+            <%= request_url = try(:request).env["HTTP_REFERER"] || t(".unknown") %>
+          </div>
+        </div>
```

Comment:
The HTTP request method is missing from the information list as was requested at the original proposal:
https://meta.decidim.org/processes/roadmap/f/122/proposals/15923

> This information could be:
> [...]
> **HTTP request method (GET/POST/etc.)**